public/javascripts/index.js

```diff
@@ -1,3 +1,44 @@
-window.addEventListener("load", (event)=>{
-    console.log("hello from javascript!")
+window.addEventListener("DOMContentLoaded", (event)=>{
+    let searchBox = document.getElementById('search')
+    let searchButton = document.getElementById('searchButton')
+    searchButton.addEventListener('click', async (event) =>{
+        let res = await fetch(`http://localhost:8080/api/search?search=${searchBox.value}`)
+        let {questions} = await res.json()
+        let questionContainer = document.getElementById('questionsContainer')
+        questionContainer.innerHTML = ""
+        questions.forEach(question =>{
+            let container = document.createElement('div')
+            container.classList.add('question')
+
+            let link = document.createElement('a')
+            link.href = `/questions/${question.id}`
+            link.classList.add('questionLink')
+            link.innerText = `Q: ${question.title}`
+            let body = document.createElement('p')
+            body.innerText = question.body
+
+            let infoLine = document.createElement('div')
+            let answerCount = document.createElement('div')
+            answerCount.classList.add("answers")
+            if(question.Answers){
+                answerCount.innerText = "Answers: " + question.Answers.length
+            }
+            else{
```

Comment:
This `else` should be on the same line as the closing bracket of the `if` it is associated with.

Reply:
Made those changes as requested